scuba/__main__.py

```diff
@@ -366,7 +369,7 @@ def __generate_hook_script(self, name):
 
             self.add_env('SCUBAINIT_HOOK_{}'.format(name.upper()), f.container_path)
 
-            writeln(f, '#!/bin/sh')
+            writeln(f, '#!{}'.format(shell))
```

Comment:
Just noting that for the hook scripts, we definitely *do* need the she-bang line since `scubainit` simply invokes them with `system`:

https://github.com/JonathonReinhart/scuba/blob/3bb7dcda5764ec9536e5c467bc93e43a8191e645/scubainit/scubainit.c#L429-L435